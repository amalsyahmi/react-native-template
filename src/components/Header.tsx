import React from 'react';
import {SafeAreaView, Text, View} from 'react-native';
export const Header = () => {
  return (
    <SafeAreaView>
      <View style={{paddingVertical: 20}}>
        <Text style={{alignSelf: 'center', fontSize: 25, color: '#555', fontWeight: 'bold'}}>
          React Native Template
        </Text>
      </View>
    </SafeAreaView>
  );
};
