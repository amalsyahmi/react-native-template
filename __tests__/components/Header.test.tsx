import 'react-native';
import React from 'react';
import { create } from 'react-test-renderer';

import {Header} from '../../src/components/Header';

it('renders correctly', () => {
  const ui = create(<Header />).toJSON();
  expect(ui).toMatchSnapshot();
});
