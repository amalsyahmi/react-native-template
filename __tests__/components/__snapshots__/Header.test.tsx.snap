// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
>
  <View
    style={
      Object {
        "paddingVertical": 20,
      }
    }
  >
    <Text
      style={
        Object {
          "alignSelf": "center",
          "color": "#555",
          "fontSize": 25,
          "fontWeight": "bold",
        }
      }
    >
      React Native Template
    </Text>
  </View>
</RCTSafeAreaView>
`;
